include/engine/datafacade/internal_datafacade.hpp

```diff
@@ -76,7 +76,7 @@ class InternalDataFacade final : public BaseDataFacade
 
     util::ShM<util::Coordinate, false>::vector m_coordinate_list;
     util::PackedVector<OSMNodeID, false> m_osmnodeid_list;
-    util::ShM<NodeID, false>::vector m_via_node_list;
+    util::ShM<extractor::GeometryID, false>::vector m_via_node_list;
```

Comment:
It used to hold nodes a few months ago, then it held nodes _or_ references to the `m_compressed_edge_container` array. For the last few months (as of https://github.com/Project-OSRM/osrm-backend/commit/21245273b414f6034fa9e686fe46ac387fa97f8f) this list has only been references to the `m_compressed_edge_container`. I think the appropriate thing to do here would be to change the name of this list to `m_via_geometry_list`.